src/routes/Root.tsx

```diff
@@ -1,10 +1,115 @@
-import { Form } from "@lib/main";
+import { Button } from "@lib/gds-components";
+import { Link } from "react-router-dom";
+
+import styles from "./root.module.css";
 
-// The root will eventually be the starting page of the form (create or update)
 export const Root = () => {
   return (
-    <>
-      <Form />
-    </>
+    <div className="govuk-width-container app-width-container">
```

Comment:
should this be in its own component inside pages/ like the other pages?

Reply:
to be clear I am not saying to move Root component to pages/ but just to extract the content of the page inside a new component inside pages/CreateTimetablePage and then import and use it inside Root (like we were doing for Form)

Reply:
I think either move the root component or leave it? Moving the content then importing sounds like the root component would be pointless

Reply:
that way all the pages are in the same directory